.TH std::nested_exception::~nested_exception 3 "Jun 28 2014" "2.0 | http://cppreference.com" "C++ Standard Libary"
.SH Synopsis
   virtual ~nested_exception() = default;  \fI(since C++11)\fP

   Destroys the nested exception object.
